### PR TITLE
Send email after create user

### PR DESCRIPTION
**Pull request by SBlazhko**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -20,4 +20,16 @@
...
 
   config.active_record.dump_schema_after_migration = false
+
+  config.email_from = 'game_panel@gmail.com'
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address:              'smtp.gmail.com',
+    port:                 587,
+    domain:               'example.com',
+    user_name:            'email',
```
Comment (IlyaUmanets) [inline config/environments/production.rb]: We need use ENV here
`{ user_name: ENV['email_user_name'], password: ENV['email_password'] }

File: config/environments/test.rb [hunk 2]
```diff
@@ -33,6 +33,7 @@
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
+  config.email_from = 'game_panel@gmail.com'
```
Comment (IlyaUmanets) [inline config/environments/test.rb]: Same here

File: app/mailers/user_mailer.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class UserMailer < ApplicationMailer
+  def welcome_email(user, password)
+    @user = user
+    @user_password = password
+    mail(to: @user.email, subject: 'Welcome to Game Panel')
```
Comment (IlyaUmanets) [inline app/mailers/user_mailer.rb]: Use I18n for static text

Comment (SBlazhko) [inline app/mailers/user_mailer.rb]: fixed

File: config/environments/production.rb (hunk 1 above)
Comment (SBlazhko) [inline config/environments/production.rb]: fixed

File: config/environments/test.rb (hunk 2 above)
Comment (SBlazhko) [inline config/environments/test.rb]: fixed

File: config/application.example.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+gmail_username: 'no-reply@gmail.com'
+
+production:
+  gmail_username: <%= ENV["gmail_username"] %>
```
Comment (IlyaUmanets) [inline config/application.example.yml]: No, we shouldn't do that.
Think about it
```yml
test:
  some_message: 'Hello world'
```

Then in the app we can use it like `ENV['some_message']`

And I think for production we shouldn't write the value. Leave them blank.

File: config/application.rb [hunk 5]
```diff
@@ -8,5 +8,6 @@ module GamePanel
   class Application < Rails::Application
     config.secret_key_base = ENV['SECRET_KEY_BASE']
     config.autoload_paths << Rails.root.join('app/null')
+    config.action_mailer.default_url_options = { host: 'https://game-panel.herokuapp.com/' }
```
Comment (IlyaUmanets) [inline config/application.rb]: We should use ENV here as well, so `ENV['mailer_host']`

Comment (as-sergey-b) [inline config/application.rb]: fixed

File: config/application.example.yml (hunk 4 above)
Comment (as-sergey-b) [inline config/application.example.yml]: fixed

**Pull request merged**